std/string.d

```diff
@@ -6857,6 +6891,13 @@ S[] outdent(S)(S[] lines) @safe pure if (isSomeString!S)
     });
 }
 
+@safe pure unittest
+{
+    import std.exception: assertThrown;
```

Comment:
are you sure ? Theorically spaces are after the punctuation... e.g `foo(1, 2, 3)`.